docs/guides/data/storing_output.md

```diff
@@ -1,83 +1,87 @@
+
+!!! tip "Current Working Directory"
+    Current Working Directory (`/floyd/home/`) is the only persistent folder for CLI Jobs and Workspace. It is set as the default working directory for Job and Workspace.
+
+Anything saved in this directory at the time a job or a Workspace finishes will be preserved
 and can be accessed and reused later.
 
-The most common thing users save is model checkpoints, but anything that ends
-up in the `/output` directory at the end of a job will be saved (use your
-imagination!).
+The most common thing users save is model checkpoints, but anything that ends up in the current working directory and its subfolders at the end of a job or a workspace session will be saved (use your imagination!).
 
 Let's work through a couple of examples to see how to save data during a job.
 
 #### Example 1
 This job prints the string "Hello, world!", and saves it to a file called
-`hello.txt`. Because `hello.txt` is located in the `/output` directory, it will
+`hello.txt`. Because `hello.txt` is located in the current working directory, it will
 be saved and available after the job finishes:
 
 ```bash
 $ floyd init my_awesome_hello_world_project
-$ floyd run "echo 'Hello, world!' > /output/hello.txt"
+$ floyd run 'echo "Hello, world!" > hello.txt'
+Creating project run. Total upload size: 691.0B
 Syncing code ...
 ```
 
 !!! note
     If you are not familiar with what
-    `echo 'Hello, world!' > /output/hello.txt` does, here's a quick
+    `echo 'echo "Hello, world!" > hello.txt` does, here's a quick
     explanation:
 
-    - The `echo 'Hello, world!'` part outputs the string `Hello, world!`.
-    - The `>` part of the command redirects the printed output of `echo 'Hello, world!'`
+    - The `echo "Hello, world!"` part outputs the string `Hello, world!`.
+    - The `>` part of the command redirects the printed output of `echo "Hello, world!"`
       (which is, of course, `Hello world!`) to the file specified after the `>`.
-    - The `/output/hello.txt` part of the command specifies where the `Hello,
-      world!` should be written to: `/output/hello.txt`. Because `hello.txt` is
-      in the `/output` directory, it will be preserved for future reference and
+    - The `hello.txt` part of the command specifies where the `Hello,
+      world!` should be written to: `hello.txt`. Because `hello.txt` is
+      created inside the current working directory, it will be preserved for future reference and
       use.
 
+Here's the file in the **Files** tab:
+
+![hello world output result](../../img/storing_output/hello_world_storing_output.png)
+
+If you want to try on Workspace, create and launch a new workspace, open the Terminal, run `echo "Hello, world!" > hello.txt` and stop the Workspace. Then you will be able to retrieve instanteniously the `hello.txt` file from the Workspace's **File Viewer** as showed in this picture:
```

Comment:
If you want to try this in a Workspace:
- create and launch a new workspace
- open the Terminal
- run `echo "Hello, world!" > hello.txt`
- stop the Workspace.

Then you can view the `hello.txt` file from the workspace page: